crates/client/rpc-core/src/lib.rs

```diff
@@ -3,6 +3,7 @@
 //! Starkware maintains (a description of Starknet API)[https://github.com/starkware-libs/starknet-specs/blob/master/api/starknet_api_openrpc.json] using the openRPC specification.
 //! This crate uses `jsonrpsee` to define such an API in Rust terms.
 
+use blockifier::execution::contract_class::ContractClass;
```

Comment:
It should be returning the type specified by the [starknet-specs](https://gist.github.com/phklive/1b81127fce118f30872850dda899ef8e#file-index-rs-L262)

Reply:
@EvolveArt, I tried going down that path at the start but not too sure about the starknet-specs: for [`DEPRECATED_CONTRACT_CLASS `](https://github.com/starkware-libs/starknet-specs/blob/master/api/starknet_api_openrpc.json#L2095), what should be base64 encoded? The whole serialized `Program` field from blockifier `ContractClass`? Tagging @tdelabro for guidance

Reply:
nvm, got [it](https://github.com/eqlabs/pathfinder/blob/main/crates/rpc/src/v02/types/class.rs#L52), will update.